### PR TITLE
Nit visual changes to article template

### PR DESCRIPTION
**Pull request by wassgha**
### Changes
- Makes the heading's black highlight and read more button a bit more symmetric
- Gives a better initial cap letter.

Before:

<kbd>screen shot 2017-07-11 at 9 13 59 pm</kbd>

After:

<kbd>screen shot 2017-07-11 at 9 13 46 pm</kbd>

Before:

<kbd>screen shot 2017-07-11 at 9 10 02 pm</kbd>

After:

<kbd>screen shot 2017-07-11 at 8 53 50 pm</kbd>

### CODE REVIEW

**Comment by camelburrito** [comment]
@wassgha - we habe a ampstart-dropcap component to get the effect you have, initial cap was intentionally like that

File: components/images/fullpage-hero.css [hunk 1]
```diff
@@ -22,7 +22,7 @@
 .ampstart-image-fullpage-hero .ampstart-image-credit {
   box-decoration-break: clone;
   background: var(--color-primary);
-  padding: 0 var(--space-2) 0.2rem;
```
Comment (camelburrito) [inline components/images/fullpage-hero.css]: this padding was the desired padding per our original mocks , may be a good thing to run this thru @spacedino

File: components/images/fullpage-hero.css [hunk 2]
```diff
@@ -52,10 +52,12 @@
   display: block;
   content: '⌄';
   font-size: var(--h2);
+  margin-top: 0.5rem;
```
Comment (camelburrito) [inline components/images/fullpage-hero.css]: why do we need this? we add a spacer via the navbar css if one exists.

File: components/images/fullpage-hero.css [hunk 3]
```diff
@@ -52,10 +52,12 @@
...
 }
 
 .ampstart-readmore-text {
   background: var(--color-primary);
+  padding: 0.5rem;
```
Comment (camelburrito) [inline components/images/fullpage-hero.css]: i think this should match what is in the image credit class

File: css/ampstart-base/elements/initialcap.css [hunk 4]
```diff
@@ -23,5 +23,8 @@
   color: var(--color-font-secondary);
   font-size: var(--h1);
   font-weight: var(--bold-font-weight);
-  margin-left: -2px;
```
Comment (camelburrito) [inline css/ampstart-base/elements/initialcap.css]: This is intentional - you need to be looking at dropcap for the effect you want !

File: components/images/fullpage-hero.css (hunk 1 above)
Comment (spacedino) [inline components/images/fullpage-hero.css]: @wassgha the "Read more" looks fine, so if we want the same effect for the Title Text (which we do) than the black needs to extend above the text, below the text, to the left, and right, the same amount.  Right now its 18px from the top / bottom, and 8px from the right / left of the text. Before was actually closer : 11px from the top /bottom, and 18px from the right / left. I recommend 16px top / bottom and 16px left / right padding round the text. It's divisible by 8px, which is the system our type size is built on, so it should look proportionally the best.  Does that make sense?

//cc @camelburrito also this PR name should be more descriptive...? Is this referencing an issue?

File: components/images/fullpage-hero.css (hunk 2 above)
Comment (wassgha) [inline components/images/fullpage-hero.css]: This only applies to the downward-facing arrow under the "read more" to compensate for the added padding

File: components/images/fullpage-hero.css (hunk 1 above)
Comment (camelburrito) [inline components/images/fullpage-hero.css]: I think @wassgha - came up with this. There is no issue on this part of the template?

Comment (wassgha) [inline components/images/fullpage-hero.css]: Yes, no issue. These were just the changes I made for my Show & Tell demo :)

**Pull request closed without merging**